src/Features/Skylighting.cpp

```diff
@@ -229,7 +225,7 @@ void Skylighting::Prepass()
 				((int)cellID.x - probeArrayDims[0] / 2) % probeArrayDims[0],
 				((int)cellID.y - probeArrayDims[1] / 2) % probeArrayDims[1],
 				((int)cellID.z - probeArrayDims[2] / 2) % probeArrayDims[2],
-				(uint)settings.MaxFrames },
+				64 },
```

Comment:
If you're forcing 255 frames why is this still 64?

Also, if this should be a constant then just set it in shader.

Reply:
I doesn't fade in as fast as it should, so more work seems to be required.

Reply:
Since you already have a forceFrame, why not just adding one frame every 1/30 sec? Also current timer setup will make the actual update rate slightly slower than 30fps, since there are delays.

But yeah, 30fps update to 255 frames is a whole 8-9 second. You can't avoid that.

Reply:
After loading in it's fine. We just want to update every frame until the skylighting is finished, when loading into a new area. When moving around the world it's fine because it only updates in the distance.

Reply:
Yes but i think it's still a bit slow with jogging speed.